modules/virt-node-network-config-console.adoc

```diff
@@ -1,5 +1,5 @@
 :_mod-docs-content-type: CONCEPT
 [id="virt-node-network-config-console_{context}"]
 = Managing policy from the web console
-You can manage the policy from the web console by accessing the list of created policies in *NodeNetworkConfigurationPolicy* page under *Networking* menu. This page enables you to create, update, monitor, and delete the policies.
-
+You can update the node network configuration, such as adding or removing interfaces from nodes, by applying `NodeNetworkConfigurationPolicy` manifests to the cluster.
+Manage the policy from the web console by accessing the list of created policies in *NodeNetworkConfigurationPolicy* page under *Networking* menu. This page enables you to create, update, monitor, and delete the policies.
```

Comment:
Might be my ignorance, but you talk about a _network configuration_, _manifests_, a _policy_, and a _list of policies_. It's not clear to me which is what.

Reply:
```suggestion
Manage the policy from the web console by accessing the list of created policies in the *NodeNetworkConfigurationPolicy* page under the *Networking* menu. This page enables you to create, update, monitor, and delete the policies.
```

Reply:
I picked this up from previous writer just adding a simple line but I will look into making this improvement if it does not make sense as written